react/components/SKUSelector/README.md

```diff
@@ -1,73 +1,71 @@
-# SKU Selector
+# SKUSelector
 
-SKU Selector is a canonical component that any VTEX app can import.
+## Description
 
-To import it into your code: 
-```js
-import SKUSelector from 'vtex.store-components/SKUSelector'
-```
+`SKUSelector` is a VTEX Component that is resposible to handle events of sku selection for a product. This component can be imported and used by any VTEX App.
```

Comment:
Since we're inside the store-components app, that is not necessary and would be redundant. All the other components in this repository are using the template I followed.